HeRoBackEnd/Controllers/CandidateController.cs

```diff
@@ -14,18 +17,32 @@ public CandidateController()
           //  CandidateService candidateService = new CandidateService();
         }
 
+        /// <summary>
+        /// Returns a Json string of the list of candidates
+        /// </summary>
+        /// <returns>Json string representing a list of Candidates</returns>
+        [HttpPost]
+        [Route("Candidate/Index")]
```

Comment:
You've set routing for this controller to [controller] so its 'candidate'. And this action is at [controller]/candidate/index = candidate/candidate/index. Two much candidates ;)